R/board_connect.R

```diff
@@ -79,15 +79,16 @@
 #' # Download a shared dataset
 #' board |> pin_read("timothy/mtcars")
 #' }
-board_connect <- function(auth = c("auto", "manual", "envvar", "rsconnect"),
-                          server = NULL,
-                          account = NULL,
-                          key = NULL,
-                          cache = NULL,
-                          name = "posit-connect",
-                          versioned = TRUE,
-                          use_cache_on_failure = is_interactive()) {
-
+board_connect <- function(
```

Comment:
Sorry for the reformatting, this was just format on save with `air`. I can back this out if you want a tighter diff. Can also make a separate PR to `air` everything if you'd like.

Reply:
https://github.com/rstudio/pins-r/pull/885, if you want we could merge that first and I can rebase out these here.